src/dialects/ibmi/query-generator-typescript.ts

```diff
@@ -27,7 +27,7 @@ export class IBMiQueryGeneratorTypeScript extends AbstractQueryGenerator {
       'LEFT JOIN QSYS2.SYSCST',
       'ON QSYS2.SYSCSTCOL.CONSTRAINT_NAME = QSYS2.SYSCST.CONSTRAINT_NAME',
       'WHERE QSYS2.SYSCOLUMNS.TABLE_SCHEMA =',
-      table.schema !== '' ? `${this.escape(table.schema)}` : 'CURRENT SCHEMA',
+      table.schema ? `${this.escape(table.schema)}` : 'CURRENT SCHEMA',
```

Comment:
Same note in a few other places in this file, unnecessary concatenation :)

```suggestion
      table.schema ? this.escape(table.schema) : 'CURRENT SCHEMA',
```